source/Util.js

```diff
@@ -0,0 +1,49 @@
+const Constant = require('./Constant');
+
+function getRandomNumber(low, high, options) {
+  low = low || Constant.MIN_INTEGER;
```

Comment:
Consider using js native limits as opposed to custom ones, like for eg- `Number.MAX_SAFE_INTEGER`